incubator/etcd/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: etcd
 home: https://github.com/coreos/etcd
-version: 0.4.1
+version: 0.4.2
```

Comment:
I know this is major version 0, but shouldn't this still be a MINOR bump since functionality is added?

Reply:
Thanks! Have bumped this!